src/web/components/ClientSideTokenGeneration/CstgDomainItem.tsx

```diff
@@ -41,7 +40,7 @@ export function CstgDomainItem({
       <td className='domain'>{domain}</td>
       <td className='action'>
         <div className='action-cell'>
-          <EditButton onClick={onEditDialogChange} />
+          <ActionButton onClick={onEditDialogChange} className='action-button' icon={ActionButtonIcon.edit}/>
```

Comment:
Could we possibly move the 'action-button' styling to `ActionButton.scss`? Then we won't need to specify the `className` every time. e.g. I noticed the `ActionButton` in `ApprovedParticipantItem.tsx` is missing the className, so is styled a bit differently.